### ISSUE TITLE
opaque identifiers for some individuals

### ISSUE THREAD

**Issue by eroux** [buda-base/owl-schema#101]
I think some individuals defined in the owl file could benefit from an opaque name, for instance instead of `bdr:PlaceTypePilgrimageSite` could be renamed `bdr:PlaceType0001`. I would extend that to the lineages, the licenses (I think `bdr:LicenseCopyrighted` is not a good name), the traditions and the non-standard scripts... The main reason is that the English names we currently have may not be the most appropriates and having a supposedly opaque URI being some erroneous English name is not very satisfying... wdyt?

**Comment by xristy** [comment]
I agree w.r.t. ```:PlaceType```. I'm curious about ```:Tradition``` and ```:Lineage```. Do these concepts occur outside the Tibetan domain or should we have ```:TibetanTradition``` and ```:TibetanLineage```?

I'm agnostic about ```adm:License``` in general since they seem to be reasonably rendered in English. Is the specific issue w/ ```bdr:LicenseCopyrighted``` that it would be better named ```bdr:LicenseCopyright``` or is it that the comment says:

> Use it for works under copyright that do not say anything about the copyrights or saying All Rights Reserved.

which is technically not an assertion of (c). This brings to mind whether it is relevant to record under what jurisdiction a (c) is asserted.

**Comment by eroux** [comment]
ok. I think `bdr:LicenseCopyrighted` would be best renamed as `bdr:LicenseAllRightsReserved`. Technically a work under CC-BY is also under copyright, it's just that some of the copy rights are given to the public / waived under some conditions.

**Comment by xristy** [comment]
I'll rename to ```bdr:LicenseAllRightsReserved```.

**Comment by xristy** [comment]
I renamed ```:PlaceType``` jndividuals to opaque ids. I'll wait on using opaque ids on ```:TibetanTradition``` and ```:TibetanLineageType``` until at least the ```bo-x-ewts`` are entered on this items
